src/readii/feature_extraction.py

```diff
@@ -164,7 +329,8 @@ def radiomicFeatureExtraction(
 	keep_running: bool = False,
```

Comment:
Do you want an Optional here?
```suggestion
	keep_running: Optional[bool] = False,
```

Reply:
would it be to indicate that users dont have to pass that parameter in? 

I figure since by default its `False`, that users dont have to consider it. 
Optional would allow a `[True | False] | None ` which could be confusing i think 
- shouldnt cause any issues since `None` would satisfy a boolean check

Reply:
I guess I thought that you had to always put Optional if a default value is set.